tests/test_modeling_tf_common.py

```diff
@@ -136,6 +136,25 @@ def run_in_graph_mode():
             outputs = run_in_graph_mode()
             self.assertIsNotNone(outputs)
 
+    def test_forward_signature(self):
+        config, _ = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            model = model_class(config)
+            signature = inspect.signature(model.call)
+            # signature.parameters is an OrderedDict => so arg_names order is deterministic
+            arg_names = [*signature.parameters.keys()]
+
+            if model.config.is_encoder_decoder:
+                expected_arg_names = [
+                    "inputs",
+                    "attention_mask",
+                    "decoder_input_ids",
+                    "decoder_attention_mask",
+                    "encoder_outputs",
+                ]
+                self.assertListEqual(arg_names[:5], expected_arg_names)
```

Comment:
In TF I only check Seq2Seq models because not even all models call their first input `inputs`, but some call them `input_ids`. @sgugger @LysandreJik - do you guys think it's worth "correcting" Funnel and Elektra to call the first input argument `inputs` like the other models or not? It's also a pretty big breaking change so not left it out for now as well.

![Screenshot from 2020-09-30 21-05-49](https://user-images.githubusercontent.com/23423619/94729156-94515000-0361-11eb-99b5-a6ad5cd81735.png)

Reply:
I think we can switch Funnel since it's very recent and probably not too used right now. Electra seems more breaking but at the same time, this looks kind of necessary (sorry I used Electra as a template for Funnel...)

Reply:
Just noticed that I can depreciate `input_ids` to `inputs` which I think is the best option

Reply:
Okey added a depreciation warning and changed to `inputs` => should be good now